GithubProfileWiki/Services/FollowersAPI.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  FollowersService.swift
+//  GithubProfileWiki
+//
+//  Created by ilter on 23.01.2022.
+//
+
+import Foundation
+
+struct FollowersAPI: APIHandler {
+    var userName: String
+    
+    func submitRequest(from param: [String: Any]) -> URLRequest? {
+        
+        let urlString = Path(userName: userName).path
+        if var url = URL(string: urlString) {
+            if param.count > 0 {
```

Comment:
do you really need that check?

Reply:
Thx, if it is zero compactMap wont work and nothing changes :)